package.json

```diff
@@ -10,13 +10,14 @@
         "lerna:publish": "yarn install; lerna run clean; lerna run build; lerna publish --registry=https://registry.npmjs.org/"
     },
     "config": {
-      "mnemonic": "concert load couple harbor equip island argue ramp clarify fence smart topic"
+        "mnemonic": "concert load couple harbor equip island argue ramp clarify fence smart topic"
     },
     "devDependencies": {
-        "lerna": "^2.5.1",
+        "@0xproject/utils": "^0.1.0",
```

Comment:
How is `utils` used by this `package.json`?

Reply:
in the `postpublish_utils.js` script